### PR TITLE
Converting Cyrillic to URL and fix Russian topic title (ISO 9), Russian language

### PR DESCRIPTION
**Pull request by 0niel**
image
ApplicationFrameHost_wn7MHgjYiL

image
image

### CODE REVIEW

File: modules/Forum/classes/Forum.php [hunk 1]
```diff
@@ -604,17 +604,35 @@ public function getPosts($tid = null){
 		}
 		return false;
 	}
-
+	
+	// Converting Cyrillic to Latin letters (https://en.wikipedia.org/wiki/ISO_9)
```
Comment (samerton) [inline modules/Forum/classes/Forum.php:608]: Could this be moved to the file core/classes/Util.php and made static please? Would be easier for other modules to access this function 👍

File: custom/languages/Russian/time.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ *	Made by Samerton
+ *  https://github.com/NamelessMC/Nameless/
+ *  NamelessMC version 2.0.0-dev
+ *
+ *  License: MIT
+ *
+ *  Russian Language - Time
+ */
+
+$language = array(
```
Comment (samerton) [inline custom/languages/Russian/time.php:12]: Were you able to handle plurals correctly in translations such as this? There was a feature added over a year ago that should help if not: https://github.com/NamelessMC/Nameless/issues/1419#issuecomment-475904785

Comment (0niel) [reply custom/languages/Russian/time.php]: Yes, it works great, I added it to my translation

**Comment by Aberdeener** [comment]
Lgtm but let's wait for Sam

**Comment by 0niel** [comment]
99.8% of phrases have been translated into Russian 🥳

**Comment by samerton** [comment]
Thanks again 👍

**Pull request merged**